lib/screen/interactive_postcard/hand_signature_page.dart

```diff
@@ -196,7 +243,7 @@ class _HandSignaturePageState extends State<HandSignaturePage> {
     return metadataFile;
   }
 
-  Future<void> _handleSaveButtonPressed() async {
+  Future<void> _saveStampAndContinue({bool addSignature = true}) async {
     setState(() {
```

Comment:
I think we dont need this setState.

Reply:
we need setState instead of AsyncButton to disable other button while one async button is running